### ISSUE TITLE
whisper-medusa Support

### ISSUE THREAD

**Issue by razvanab** [chidiwilliams/buzz#877]
Can you please add whisper-medusa?
https://github.com/aiola-lab/whisper-medusa

Thank you.

**Comment by raivisdejus** [comment]
@razvanab I do not think we will be implementing this in the near future. Mainly I do not see the benefit. Quality of the transcripts in medusa approach seems a bit worse. For speed, I doubt it will be much faster than whisper.cpp on the CPU and on GPU speed does not matter that much.

Thanks for bringing this up, maybe we return to this some time later when there is more time.
